core/src/Cache/Items/MsalAccountCacheItem.cs

```diff
@@ -90,9 +94,19 @@ internal MsalAccountCacheItem(string environment, MsalTokenResponse response, st
         [DataMember(Name = "authority_type")]
         internal string AuthorityType { get; set; }
 
+        [DataMember(Name = "client_info")]
+        internal string ClientInfoString
+        {
+            get => RawClientInfo;
+            set
+            {
+                RawClientInfo = value;
+            }
+        }
+
         internal MsalAccountCacheKey GetKey()
         {
-            return new MsalAccountCacheKey(Environment, TenantId, HomeAccountId, LocalAccountId);
+            return new MsalAccountCacheKey(Environment, TenantId, HomeAccountId, PreferredUsername);
```

Comment:
>PreferredUsername [](start = 81, length = 17)

Why is this changing to PreferredUsername? Will that impact existing cache (pre-next release)?

Reply:
this change was requested by iOS team, purpose is to optimize cache lookups, 
it will not affect .NET Msal lookup because it does not perform lookup on individual keys